solutions/usaco-966.mdx

```diff
@@ -47,4 +47,48 @@ ans += first8[N % 8]
 print(ans)
 ```
 </PySection>
+<JavaSection>
+  
+```java
+
+import java.util.*;
+import java.io.*;
+
+
```

Comment:
```suggestion
```
spacing

Reply:
emmm... this one is actually the only one not resolved yet...

Reply:
Done